src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -319,6 +324,14 @@ export class ServiceFormComponent extends CdForm implements OnInit {
         this.serviceType = params.type;
       });
     }
+
+    this.cephServiceService.list().subscribe((services: CephServiceSpec[]) => {
```

Comment:
this will fetch all the services right? which can create some load if there are many services. what about just fetching for a single service. Just poll to the backend using a serviceName and if it returns successfully then the service exists, otherwise we it doesn't exists. we have this behaviour in the rgw forms and in the user-creation-form too. https://github.com/ceph/ceph/pull/34587/files

Reply:
This is not a new api call, not added in this PR, I have just reused this.